### ISSUE TITLE
Failed building

### ISSUE THREAD

**Issue by adagj** [NorESMhub/noresm-exp#16]
Hi @annefou ,
I updated links in all pages and now the building fails, but I'm not sure why and what to change. Too bad I have to disturb you every time I make some changes...

Best  regards,
Ada

**Comment by annefou** [comment]
Can look at it next week

**Comment by MichaelSchulzMETNO** [comment]
@annefou Can you help? I attempted to correct the config.yml according to the error message on circleci. 

But now it seems to not be able to read the git repository, see here:
https://app.circleci.com/pipelines/github/NorESMhub/noresm-exp/935/workflows/eb5568ba-c08a-49bf-94f6-a5777a90b821/jobs/1729

Dont know how to resolve that.

**Comment by MichaelSchulzMETNO** [comment]
update: now I am one step further, BUT: the script doesnt find python ???

**Comment by annefou** [comment]
Yes. This repo is a bit obsolete and you should think about migrating to a more recent version of jupyter book and use github action instead of circleci.

**Comment by MichaelSchulzMETNO** [comment]
How can this migation be done? ...any recipe or suggested reading to do so?

**Comment by annefou** [comment]
- Migrate the table of content (TOC) https://jupyterbook.org/structure/toc.html
- remove circleci and switch to github action: https://jupyterbook.org/basics/create.html

**Comment by annefou** [comment]
Also you could consider usin fair digital object and rohub (Https://reliance.rohub.org).

**Comment by charlienegri** [comment]
the build with GitHub Actions would skip the Jekyll config, right? or do you want to keep that?

**Comment by MichaelSchulzMETNO** [comment]
hm - no idea - if thats not needed, no.

**Comment by annefou** [comment]
you remove it. Follow the guide on the creation of Jupyter-book. It is now very simple. The github action will take care of "everything".
